### DISCUSSION TITLE
Welcome to 2fast Discussions!

### DISCUSSION THREAD

**Discussion by jp-weber** [2fast-team/2fast#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mhellmeier** [comment]
Great and hello everybody!

My name is Malte, and I am one of the founders of 2fast together with @jp-weber. The initial idea came up in relation to a research project since there was no two-factor authenticator out there that uses local storage only (without a cloud connection) and can work on different platforms and operating systems. Therefore, we created 2fast!

Looking forward to some inspiring discussions!
